tests/lib/Repository/Helper/SchemaIdentifierExtractorTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Core\Repository\Helper;
+
+use Ibexa\Core\Repository\Helper\SchemaIdentifierExtractor;
+use PHPUnit\Framework\TestCase;
+
+class SchemaIdentifierExtractorTest extends TestCase
+{
+    public function testExtract()
```

Comment:
Please remember that new code needs to be strict:
```suggestion
    public function testExtract(): void
```